config/example-config.yaml

```diff
@@ -54,7 +54,7 @@ processors:
 
 exporters:
   qryn:
-    dsn: tcp://clickhouse-server:9000/qryn?username=qryn&password=demo
+    dsn: clickhouse://0.0.0.0:9000/qryn
```

Comment:
is this correct? 0.0.0.0 is generally used in server context to listen on all interfaces, not in clients. Also most setups will use authentication

Reply:
Reverted